assigner/backends/decorators.py

```diff
@@ -0,0 +1,13 @@
+from assigner.backends import GitlabBackend
+from assigner.config import config_context
+
+
+@config_context
+def require_backend(config, func):
```

Comment:
I'm not pleased with this line decorator. Not only does it not work right now (:D), it requires config which causes a file read. Then most of these commands require config too before this was called. I can see us opening and closing config 3 or 4 times for a single function call.

I'm definitely open to better ideas. I really like the decorator idea... maybe we should just cache a readonly config on the decorator object?

cc @LinuxMercedes